src/store/reducers/searchParams.js

```diff
@@ -0,0 +1,30 @@
+import actionTypes from '../../constants/actions';
+import { parseSearchParams } from '../../utils/searchParams';
+
+
+const getInitialState = () => {
+  const { modal } = parseSearchParams(window.location.search);
+  return { modal };
```

Comment:
the `location` object has this value already and is available throughout the app, why persist on Redux store? seems redundant.